subtests/docker_cli/systemd/p4321.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Systemd for p4321 service
+After=docker-latest.service
+
+[Service]
+Environment=NAME=p4321
+ExecStartPre=-/usr/bin/docker-latest stop p4321
+ExecStartPre=-/usr/bin/docker-latest rm -f p4321
```

Comment:
Probably not a good idea. These are only useful if p4321 is running when it isn't supposed to be, and that would be a bug.